tests/runtime/engine/main.py

```diff
@@ -72,11 +72,11 @@ def main(test_filter, run_aot_tests):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description='Runtime tests for bpftrace.')
-    parser.add_argument('--filter', dest='tests_filter',
-                        help='filter runtime tests')
+    parser.add_argument('--test-filter', type=str,
```

Comment:
I don't see a reason for renaming the option and it will require to rewrite existing scripts that people may have, so I'd prefer sticking with the old name.

Reply:
Correct. Actually we support filtering a single test right now, so thought of removing 's' from option.

Reply:
Feel free to rename the internal variable to `test_filter` but don't change the user-facing CLI please.

Reply:
To make it clear: I prefer sticking with the original CLI option name, i.e. `--filter`.